android/guava/src/com/google/common/collect/CompactHashMap.java

```diff
@@ -143,53 +140,34 @@
    */
   @VisibleForTesting @MonotonicNonNullDecl transient Object[] values;
 
-  /** The load factor. */
-  transient float loadFactor;
-
   /**
    * Keeps track of modifications of this set, to make it possible to throw
    * ConcurrentModificationException in the iterator. Note that we choose not to make this volatile,
    * so we do less of a "best effort" to track such errors, for better performance.
    */
   transient int modCount;
 
-  /** When we have this many elements, resize the hashtable. */
-  private transient int threshold;
-
   /** The number of elements contained in the set. */
   private transient int size;
 
   /** Constructs a new empty instance of {@code CompactHashMap}. */
   CompactHashMap() {
-    init(DEFAULT_SIZE, DEFAULT_LOAD_FACTOR);
+    init(DEFAULT_SIZE);
   }
 
   /**
    * Constructs a new instance of {@code CompactHashMap} with the specified capacity.
    *
    * @param capacity the initial capacity of this {@code CompactHashMap}.
```

Comment:
Oops, I believe either this `@param` should be renamed to `expectedSize` or the corresponding parameter should be named back to `capacity`. :)

Reply:
good catch!